features/0794-did-rotate/README.md

```diff
@@ -19,6 +19,10 @@ This protocol is only applicable to DIDComm v1 - in DIDComm v2 use the more effi
 This mechanism allows a party in a relationship to change the DID in use for the relationship. This may be used to switch DID methods,
 but also to switch to a new DID within the same DID method. Inspired by (but different from) the DID rotation feature of the DIDComm Messaging (DIDComm v2) spec.
```

Comment:
“new DID within the same DID method” to “new DID within the same DID method, such as to update the mediator relationship in the DIDDoc.”

I think it would be helpful to include that example.

Reply:
Super minor, and not necessary, but I realize there are other motivators for adding/using this protocol, and perhaps a list is needed:

- change mediators
- change the key in an non-updatable DID method.
- hanging up
- Switch from a deprecated DID Method, such as unqualified DIDs or `did:peer` numalogos 1 and 2/3 to `did:peer:4`